Sources/StreamChat/Controllers/ChannelListController/ChannelListController.swift

```diff
@@ -129,7 +129,7 @@ public class ChatChannelListController: DataController, DelegateCallable, DataSt
                 callback: { [unowned self] in
                     switch $0.webSocketConnectionState {
                     case .connected:
-                        self.updateChannelList(trumpExistingChannels: self.channels.count > self.requestedChannelsLimit)
+                        self.updateChannelList(trumpExistingChannels: true)
```

Comment:
AFAICT this is not a needed change, I would leave it alone

Reply:
Hm, I would have it 🤔 

When we reconnect, we should invalidate history of local channels because there can be changes we are not aware of.

Use case:
1. load 1st page (25 channels)
2. go offline
3. user becomes a member of a new channel that goes to 1st page (forcing locally existed channel X out from 1st page)
4. smth changes in channel X outside the 1st page of messages
5. go back online and do not trump local data

The history of channel X will be out of sync and user won't see the updates

Reply:
I would trump on `synchronise/reconnect` no matter the storage setup. With no `/sync` and enabled local storage, queries will go out of sync very soon